content/topics/java-specific/intro-to-gradle-with-intellij/_index.md

```diff
@@ -1,3 +1,40 @@
 ---
 title: Introduction to Gradle
 ---
```

Comment:
please add "ready: true" before "---" otherwise the contentlinks will show up as [TODO]